common-ui-compose/src/main/java/app/tivi/common/compose/SortMenuPopup.kt

```diff
@@ -39,14 +39,12 @@ fun SortMenuPopup(
     content: @Composable () -> Unit,
 ) {
     var sortPopupOpen by remember { mutableStateOf(false) }
+    IconButton(
```

Comment:
Are `IconButton` and `DropdownMenu` supposed to be in a `Column` or `Row`? If they could be anywhere, shouldn't this be inside a `Box`?